virttest/ppm_utils.py

```diff
@@ -26,6 +26,12 @@
                     'and Windows guest BSOD detection are disabled. '
                     'In order to enable it, please install python-imaging or '
                     'the equivalent for your distro.')
+# Prevent logs pullotion
```

Comment:
Typo: pollution

Reply:
Good catch, just found it is wrong in all the places..